snippets/size.md

```diff
@@ -0,0 +1,22 @@
+### size
+
+Get size of arrays, objects or strings.
+
+Get type of value – array, object and string. Use `length` property for arrays. Return `length` or `size` value if available or number of object keys. Split strings into array of characters with `split('')` and return its length.
+
+```js
+const size = value =>
+  Array.isArray(value)
+    ? value.length
+    : value && typeof value === 'object'
+    ? value.size || value.length || Object.keys(value).length
+    : typeof value === 'string'
+    ? value.split('').length
```

Comment:
@atomiks mentioned this but didn't start a review. should just use `value.length` to avoid the loop and because both values should be the same.

Problem: Emojis. While valid unicode, don't work quite right with this.

😄 has a length of 2, but a bytesize of 4

Reply:
I honestly don't know how to highlight a section of the code 🤔 😆 . You can use the `byteSize` string's trick, `new Blob([str]).size`.